README.md

```diff
@@ -1,164 +1,35 @@
-
-The [clkhash schema files](https://clkhash.readthedocs.io/en/latest/schema.html) specify the fields that will be used in the hashing process as well as assigning weights to those fields. The `example-schema` directory contains a set of example schema that can be used to test the tools.
-
-`garble.py` will provide usage information with the `-h` flag:
-
-```
-$ python garble.py -h
-usage: garble.py [-h] --source SOURCE --schema SCHEMA --secretfile SECRET_FILE
-
-Tool for garbling PII in for PPRL purposes in the CODI project
-
-required arguments:
-  --source SOURCE  Source PII CSV file
-  --schema SCHEMA  Directory of linkage schema
-  --secretfile SECRET_FILE  Location of de-identification secret file
-
-optional arguments:
-  -h, --help       show this help message and exit
-```
-
-`garble.py` will package up the garbled PII files into a [zip file](https://en.wikipedia.org/wiki/Zip_(file_format)) called `garbled.zip`.
-
-Example execution of `garble.py` is shown below:
-
-```
-$ python garble.py --source /Users/andrewg/projecs/anonlink-multiparty/data/siblings/system-a.csv --schema /Users/andrewg/Desktop/schema
-Salt value:
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 321clk/s, mean=1.16e+3, std=21.2generating CLKs: 100%|█| 252/252 [00:00<00:00, 420clk/s, mean=1.17e+3, std=36.1]
-CLK data written to output/name-sex-dob-phone.json
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 417clk/s, mean=1.17e+3, std=22.4generating CLKs: 100%|███| 252/252 [00:00<00:00, 535clk/s, mean=1.17e+3, std=32]
-CLK data written to output/name-sex-dob-zip.json
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 340clk/s, mean=1.53e+3, std=73.1generating CLKs: 100%|█| 252/252 [00:00<00:00, 424clk/s, mean=1.54e+3, std=68.2]
-CLK data written to output/name-sex-dob-parents.json
-generating CLKs:  21%|▏| 52.0/252 [00:00<00:00, 423clk/s, mean=1.17e+3, std=23.7generating CLKs: 100%|███| 252/252 [00:00<00:00, 532clk/s, mean=1.17e+3, std=34]
-CLK data written to output/name-sex-dob-addr.json
-```
-
-## Mapping LINK_IDs to PATIDs
-
-When anonlink matches across data owners / partners, it identifies records by their position in the file. It essentially uses the line number in the extracted PII file as the identifier for the record. When results are returned from the DCC, it will assign a LINK_ID to a line number in the PII CSV file.
-
-To map the LINK_IDs back to PATIDs, use the `linkidtopatid.py` script. The script takes two arguments:
-
-1. The path to the PII CSV file
-1. THe path to the LINK_ID CSV file provided by the DCC
-
-The script will create a file called `linkidtopatid.csv` with the mapping of LINK_IDs to PATIDs.
-
-## Notice
-
-Copyright 2020 The MITRE Corporation.
-
-Approved for Public Release; Distribution Unlimited. Case Number 19-2008
+# Data Owner Tools Executables
```

Comment:
This change overwrites the existing README file. These should be added to the README instead.